src/shared/constants/show-me-questions/show-me-questions.cat-home-test.constants.ts

```diff
@@ -0,0 +1,66 @@
+import { VehicleChecksQuestion } from '../../../providers/question/vehicle-checks-question.model';
```

Comment:
Are these the correct questions and codes? As far as I was aware we didn't have the questions for HOME tests

Reply:
yes they are, the document i was referencing is in the attachments here - https://jira.dvsacloud.uk/browse/MES-5015 as it stated in the comments for this one https://jira.dvsacloud.uk/browse/MES-4918

Reply:
Ok, could you take screenshots of them in the app (the list) and ask Steve/Graham to double-check them. We have a lot of issues with the questions so written sign off is great :)

Reply:
Okay, sure thing! :)

Reply:
Graham has asked that we hang fire until tomorrow as they are finalising the list still, as a result of this regardless of whatever reviews go in today, I won't be merging this in or moving my stories to 'Done' in Jira until that has the go ahead.